### ISSUE TITLE
Sliding item: click options buttons under item

### ISSUE THREAD

**Issue by manucorporat** [driftyco/ionic#6981]
While sliding an item, the item becomes `pointer-events: none` so users can click the option buttons behind accidentally.

This is a known bug, but it is not easy to reproduce, it was fixed by in this commit https://github.com/driftyco/ionic/commit/c28aa53478771f2ffcab191a049adbc9d777bd54#diff-396ddb8f5fdb3d116cbf4643211d3654L63

but this change created a more important bug: https://github.com/driftyco/ionic/issues/6950
so i just reverted the change here: https://github.com/driftyco/ionic/commit/38ab17b0a8b91698b60d3f75420006a61b7500e1

We still need to find a way to fix this issue without breaking anything. A possible idea to add a new css class that makes `ion-item-options` `pointer-events:none` until the sliding item reaches a stable state (fully open).

**Comment by mpaland** [comment]
Is it possible to add ionOpen and ionClose events to the sliding item and intercept these in the background to modify the item class?

**Comment by manucorporat** [comment]
@mpaland do you mean to fix this issue?

**Comment by mpaland** [comment]
Right, this was meant as possible solution and referred to your last paragraph above.

**Comment by aggarwalankush** [comment]
@manucorporat seems like this issue is in backlog because of problems you explained. It's been here from June, don't let it die in backlog. I hope you will resolve it soon. 👍
